### DISCUSSION TITLE
Sugerencia de traducción

### DISCUSSION THREAD

**Discussion by cjosue15** [reactjs/es.reactjs.org#571] [Q&A] [answered]
Tengo una consulta sobre como se podria traducir esta oración:

"Any other values, such as strings, numbers, or arbitrary objects and arrays, are not React elements."

¿Estaría bien así?

"Cualquier otro valor, como strings, numeros u objetos arbitrarios y `arrays`, no son elementos de React."

**Comment by carburo** [comment, marked as answer]
Sí, creo que quedaría bien así. Solo notar que los extranjerismos los solemos señalar en esta documentación con cursiva (_arrays_, no `arrays`), aunque en palabras de uso bastante generalizado también podemos optar por mantenerlas en redonda:

«Cualquier otro valor, como _strings_, números u objetos arbitrarios y _arrays_, no son elementos de React».
